package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "Microverse project",
+  "private": true,
```

Comment:
Description may have the projects actual name instead of "Microverse Project" it may read " Todo List App"

Reply:
Thank you @AbuTalha3 for this catch! I've changed the description :)